public/send-results.js

```diff
@@ -0,0 +1,26 @@
+/* global: location, sessionStorage */
+
+(function() {
+  var path = location.pathname.slice(1)
+
+  if (path === "login") {
+    var submitSurveyButton = document.getElementById('submit-survey-button')
+    var clinicianEmailInput = document.getElementById('clinician-email-input')
+    submitSurveyButton.addEventListener('click', function() {
+      var storeClinicianEmail = sessionStorage.setItem('clinician-email', clinicianEmailInput.value)
+    })
+  }
+
+  if (path === "results") {
+    var sendResultsButton = document.getElementById('send-results-button')
+    var getSessionStorage = window.sessionStorage
+    var getClinicianEmail = getSessionStorage['clinician-email']
+    var emailSubject = getSessionStorage['select-survey'] + ' results ' + Date()
+    var emailBody = JSON.stringify(getSessionStorage)
```

Comment:
Hmm. Can you do this? I can not find this documented anywhere, and trying it in a repl fails..